test/torch/tensors/test_precision.py

```diff
@@ -37,6 +37,20 @@ def test_fix_prec_registration(hook):
         assert hook.local_worker.get_obj(x.id) == x
 
 
+def test_fixed_precision_mod_operation():
+    # Test mod operation with scalar
+    x = torch.tensor([1, 2, 3]).fix_prec()
+    y = x % 3
+    y = y.float_prec()
+    assert (y == torch.tensor([1.0, 2.0, 0.0])).all()
```

Comment:
Q: Could you also add tests if the operators are ```AST``` and ```FPT```?

Reply:
I tried this, but the tests fail and I don't know how to do a mod operation where AST is the dividend and FTP the divisor... Does this work when both the dividend and the divisor are ASTs?

Reply:
For instance, consider the following: 

```python
x = torch.tensor([1, 2, 3]).share(bob, alice, crypto_provider=james)
(x % 3).get() 
```

This turns out to be `tensor([1, 2, 3])`, which is not right 😕 ❗ ❓

In fact, it's inconsistent: sometimes I get something like `tensor([4, 2, 3])`. What does this mean? 😱

Reply:
Lol - never mind. I forgot to use fixed precision I think...

Reply:
I saw the tests [here](https://github.com/OpenMined/PySyft/blob/aafa70d096acf22d7754da8b7b0037f74d12313a/test/torch/tensors/test_additive_shared.py#L1045) for the mod operation on ASTs and tried to replicate it somehow. Let me know what you think.